service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -178,52 +180,72 @@ private void configureMasProcessing() {
         .to(END_POINT_RFD)
         .otherwise()
         // Off ramp if the Sufficient For Fast Tracking is null
-        .to(ENDPOINT_ACCESS_ERR)
+        .setProperty("offRampError", constant("Sufficiency cannot be determined."))
+        .setProperty("sourceRoute", constant("assessorError"))
+        .log("Assessor Error. Off-ramping claim")
+        .process(masAccessErrProcessor)
+        .to(ENDPOINT_OFFRAMP_ERROR)
         .end();
 
     String rfdRouteId = "mas-rfd";
+    String pdfFailError = "docUploadFailed";
     from(END_POINT_RFD)
         // input: MasAutomatedClaimPayload
         .routeId(rfdRouteId)
         .wireTap(ENDPOINT_AUDIT_WIRETAP)
         .onPrepare(auditProcessor(rfdRouteId, "Sufficient evidence for ready for decision."))
         // Upload PDF
+        .doTry()
         .to(ENDPOINT_UPLOAD_PDF)
-        // Check and update claim
+        .doCatch(BipException.class)
+        // Completion code needs the MasProcessingObject as the body.
+        .setBody(simple("${exchangeProperty.payload}"))
+        .setProperty("offRampError", constant(pdfFailError))
+        .setProperty("sourceRoute", constant(rfdRouteId))
+        .to(ENDPOINT_OFFRAMP_ERROR)
+        .stop()
```

Comment:
stop is now explicit to ensure that the last masComplete after the catch doesn't run.

Reply:
This should also allow future ENDPOINT_MAS_COMPLETE error handling to stand independently of this try catch as needed.